zygrader/ui/components.py

```diff
@@ -669,6 +671,9 @@ def set_searchable(self, prompt: str, search_fn: Callable):
     def set_sortable(self):
         self._sortable = True
 
+    def set_paged(self):
+        self._paged = True
```

Comment:
It feels weird to make this irreversible, even if there aren't any current use cases for enabling paging and subsequently undoing it.
```suggestion
    def set_paged(self, paged=True):
        self._paged = paged
```

Reply:
Looking everywhere, we would want to do this in Layers as well, with the default param there, so we could make this a requried param here.
```suggestion
    def set_paged(self, paged):
        self._paged = paged
```

Reply:
Fair enough. I can't imagine any case where we would want to change it though

Reply:
Yeah, I started turning that way by the end of looking at this, and even more so now think that the suggestion above doesn't actually make sense to add.